docs/events/timeline-ui-overview.asciidoc

```diff
@@ -202,7 +202,25 @@ The {ref}/esql.html[Elasticsearch Query Language ({esql})] provides a powerful w
 
 You can use {esql} in Timeline by opening the **{esql}** tab. From there, you can: 
 
-- Explore your events using the default query, or create a custom one. The default query searches documents within the Security alert index (`.alerts-security.alerts-default`) and indices specified in the <<update-sec-indices,Security data view>>, then returns 10 events from the defined time range.  
+- Write an {esql} query to explore your events. For example, start with the following query, then iterate on it to tailor your results: 
++
+[source,esql]
+----
+FROM .alerts-security.alerts-default,apm-*-transaction*,auditbeat-*,endgame-*,filebeat-*,logs-*,packetbeat-*,traces-apm*,winlogbeat-*,-*elastic-cloud-logs-*
+| LIMIT 10
+| KEEP @timestamp, message, event.category, event.action, host.name, source.ip, destination.ip, user.name
+----
++
+This query does the following:
+
+** It starts by querying documents within the Security alert index (`.alerts-security.alerts-default`) and indices specified in the <<update-sec-indices,Security data view>>.
+** Then, the query limits the output to the top 10 results.
+** Finally, it keeps the default `@timestamp`, `message`, `event.category`, `event.action`, `host.name`, `source.ip`, `destination.ip`, and `user.name` Timeline fields in the output.
```

Comment:
```suggestion
** Finally, it keeps the default `@timestamp`, `message`, `event.category`, `event.action`, `host.name`, `source.ip`, `destination.ip`, and `user.name` fields in the output.
```
Not sure it makes sense to call them Timeline fields instead of just fields

Reply:
While I think both versions work, I feel like the original is slightly more accurate and descriptive. Similar to the default fields that display on the rule details page, we also have default fields that display in Timeline (example below):


<img width="800" alt="Screenshot 2024-01-10 at 11 54 28 AM" src="https://github.com/elastic/security-docs/assets/79124755/a01ed052-8720-4abd-b9d4-d9a9f3c4da48">

If I moved "Timeline" closer to "default", would that be more clear?

```suggestion
** Finally, it keeps the default Timeline fields (`@timestamp`, `message`, `event.category`, `event.action`, `host.name`, `source.ip`, `destination.ip`, and `user.name`) in the output.
```